ge_worker/_start_bonuses_worker.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+celery -A bonuses worker --loglevel=INFO
```

Comment:
Do we need both the `beat` one and this one?

Reply:
In terms of functionality yes one starts the worker, the other starts  the beat scheduler, but in terms of scripts , I could run them from them same script, again this was a choice in view of being consistent with the Sempo build.

Reply:
Sorry the question came from ignorance, I read up a bit and apparently both are needed for this.